docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,68 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |filter contacts by name, number, address, email or tag |find specific contacts easily
+
+|`* * *` |basic user |add display pictures to my contacts |easily identify people with the same name
+
+|`* * *` |basic user |call/email my contacts by clicking on them |save time in opening my email or phoning app
+
+|`* * *` |basic user |be able to change the theme of the app from dark to light |reduce the stress on my eyes
+
+|`* * *` |basic user |customise the background picture of the app |enhance my user experience since it looks nicer
+
+|`* *` |user |edit my tags more conveniently |not need to delete all the tags when editing somebody's tags
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |basic user |be able to put tags of different colours on my contacts |identify them more easily
+
+|`* *` |new user |add more icons to the app |make it look prettier
+
+|`* *` |new user |see several interactive text fields like a template when I use the add command |not need to memorise the command for adding people
+
+|`* *` |user who uses other address book apps |import/export my address book to other format |transfer address book data between applications easily
+
+|`* *` |user who often contacts people |add Speed dial or a favourites list |easily contact those that I am in frequent touch with
+
+|`* *` |user with bad vision |adjust the font size of the address book |see the contents easily
+
+|`* *` |user who has contacts with many tags |have each tag with their own associated colour |customise my contacts better for my own visibility
+
+|`*` |user |look at a prettier address book |enhance my user experience
```

Comment:
Does this use story overlap with the user story to allow users to customize the app's theme?